jetstream/tests/test_experimenter.py

```diff
@@ -311,6 +311,49 @@
 }
 """  # noqa:E501
 
+FIREFOX_IOS_EXPERIMENT_FIXTURE = """
+{
+   "schemaVersion":"1.4.0",
+   "slug":"nimbus-aa-validation-for-ios",
+   "id":"nimbus-aa-validation-for-ios",
+   "application":"org.mozilla.ios.FirefoxBeta",
+   "appName":"firefox_ios",
+   "appId":"org.mozilla.ios.FirefoxBeta",
```

Comment:
@tdsmith  This is what the Experimenter API is currently returning. Shouldn't this be `org.mozilla.ios`?

Reply:
I think they're shipping to beta, so that seems right.